flagd/pkg/service/flag-evaluation/eventing.go

```diff
@@ -6,20 +6,28 @@ import (
 	iservice "github.com/open-feature/flagd/core/pkg/service"
 )
 
-// eventingConfiguration is a wrapper for notification subscriptions
-type eventingConfiguration struct {
+// EventingConfiguration is a wrapper for notification subscriptions
+type EventingConfiguration struct {
```

Comment:
Can you introduce a simple interface for this so that it improves the testablity of the implementation?